core/src/com/po/fuck/view/Sprites/BasicSpriteInfo.java

```diff
@@ -0,0 +1,38 @@
+package com.po.fuck.view.Sprites;
+
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.utils.JsonValue;
+
+/*
+ * assetsData.json description
+ * all textures have:
+ * each .png stores some table of frames for the object
```

Comment:
`stores some table of frames for the object` very precisely... In which order ?

Reply:
3x3:
1 2 3
4 5 6
7 8 9

Reply:
Add it to the description.